frontend/src/app/components/about/about.component.html

```diff
@@ -406,8 +406,12 @@ <h3 i18n="about.maintainers">Project Maintainers</h3>
   <div class="copyright">
     <div class="title">
       Copyright &copy; 2019-2023<br>
-      The Mempool Open Source Project
+      Mempool Space K.K.<br>
+      and other shadowy super-coders
     </div>
+    <p>
+      See our <a href="https://mempool.space/trademark-policy">Trademark Policy and Guidelines</a> for more details, published on &lt;https://mempool.space/trademark-policy&gt;.
+    </p>
```

Comment:
Not sure why you're adding this when the exact same text is below in the Trademark Notice section?
```suggestion
```